packages/shorebird_cli/lib/src/validators/shorebird_flutter_validator.dart

```diff
@@ -120,8 +120,15 @@ This can cause unexpected behavior if the version gap is wide. If you're seeing
       ['--version'],
       useVendedFlutter: !checkPathFlutter,
     );
-    final output = result.stdout.toString();
 
+    if (result.exitCode != 0) {
+      throw Exception(
+        'Flutter version check did not complete successfully.'
```

Comment:
nit but we could use a multi-line string here:

```dart
throw Exception(
'''
Flutter version check did not complete successfully.
${result.stderr}
'''
```

Reply:
Done.